src/css/styles.css

```diff
@@ -130,6 +130,12 @@
   @apply text-gray-400;
 }
 
+.navbar__logo img {
+  @apply w-[168px];
+  @apply h-10;
+  @apply -mt-2;
+}
```

Comment:
estimating of alignment + size setting might not be ideal here

Reply:
agree, but there are no place for that in the component, since we are using docusaurus component.

Reply:
the logo (red outline) content is extending out of its parent box model (blue outline). these are usually the cause for broken layouts

![Screenshot 2022-06-01 at 9 46 30 AM](https://user-images.githubusercontent.com/4774314/171312017-4702f3d4-8292-4aa1-be9a-8d4557b888fb.png)

thoughts: if there is a compulsory requirement for a fixed dimensions logo, try applying only the width to the parent box instead (leave the height), then logo max-width: 100%. if not compulsory, even better. leave it as it is

Reply:
hmm ok will fix it. was asked to increase the size to the one in figma so need to adjust it.